src/components/Navbar.jsx

```diff
@@ -0,0 +1,100 @@
+                    </div>
+                    <div className="experience-con flex-row-sb">
+                        <a className="link" href="#">
+                            <h5>Offers</h5>
+                        </a>
+                    </div>
+                    <div className="location-con">
+                        <div className="location flex-row-sb">
+                            <a className="link" href="#">
+                                <h5>Location</h5>
+                            </a>
+                            <i className="fa-solid fa-angle-down"></i>
+                        </div>
+                    </div>
+                    <div className="dropdown account-con">
+                        <button  onClick={showOptions}><i className="fa-regular fa-user text-md dropbtn"></i></button>
+                        {show && <div id="myDropdown" className="dropdown-content" style={{display: 'block'}}>
+                            <a href="/screens/sign-up.html" className="p-1">Sign Up</a>
+                            <a href="/screens/log-in.html" className="p-1">Log In</a>
+                            <a href="/screens/user-profile.html" className="p-1">My Account</a>
+                        </div> }
+                    </div>
+                    <div className="wish-con">
+                        <a href="/screens/wishlist-page.html">
+                            <span className="badge-icon badge-container"><i
+                                    className="fas fa-heart badge-icon-base text-md"></i>
+                                <span className="count-badge heart-badge-number">4</span>
+                            </span>
+                        </a>
+                    </div>
+                    <div className="cart-con">
+                        <a href="/screens/cart-page.html">
+                            <span className="badge-icon badge-container"><i className="fa-solid fa-cart-shopping text-md"></i>
+                            <span className="count-badge cart-badge-number">3</span></span>
+                        </a>
+                    </div>
+                </div>
+                <div className="bars-con" onClick={hamburgerMenu}>
+                    <div className="user">
+                        <i className="fas fa-bars"></i>
+                    </div>
+                </div>
+            </div>
+            <div className="search-btm p-1">
+                <form action="#" className="form-search flex-row-sb">
+                    <input type="text" className="text-rg pl-1" id="search-field" placeholder="search the entire store..." />
+                    <button type="submit" className="btn-icon pr-sm"><i
+                            className="fa-solid fa-magnifying-glass search-icon thin"></i></button>
+                </form>
+            </div>
+            {showHamburgerMenu && <div className="menu p-sm" style={{display: 'block'}}>
```

Comment:
Would suggest to use ternary operator instead of conditional add by writing showHamburgerMenu?the result :null 
Ternaries are more explicit in telling React what to display on the view  [Check this out](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx)

Reply:
Ohh didn't know it is a better way. Thanks for sharing. Would keep in mind!